src/supplemental-ui/css/site-extra.css

```diff
@@ -20,6 +20,20 @@ header .navbar.navbar-top {
     background-color: transparent;
 }
 
+.navbar-burger {
+    height: 3rem;
+    border-radius: 50%;
```

Comment:
With the added background color with that hover, I think it looks slightly better if we move the button over slightly:

![Uploading Screenshot 2024-02-12 at 9.57.28 PM.png…]()


```suggestion
    border-radius: 50%;
    right: 12px;
    top: 3px;
```